toolkit/tools/internal/csvparser/csvparser.go

```diff
@@ -0,0 +1,78 @@
+package csvparser
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+)
+
+var timeArray [][]string
+
+// Reads a CSV file and appends line by line to array
+func CSVToArray(filename string) {
+	file, err := os.Open(filename)
+
+	if err != nil {
+		fmt.Println("failed to open csv file")
```

Comment:
Use our logging package here (`logger.Log.Warnf("Message is '%s', "foo")`)

Reply:
Deferring this change to the next PR.